### PR TITLE
[xdg_directories] Bump version to pick up recent changes

### PR DESCRIPTION
**Pull request by gspencergoog**

### CODE REVIEW

File: packages/xdg_directories/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## [0.1.1] - Bump version to reduce dependencies on external libraries.
```
Comment (goderbauer) [inline packages/xdg_directories/CHANGELOG.md:1]: Should this just say "Reduce dependencies on external libraries" - I assume that every entry here will be a version bump?

Comment (gspencergoog) [reply packages/xdg_directories/CHANGELOG.md:1]: Sure.

**Comment by jameschensmith** [comment]
Hey @gspencergoog! Do you think you could update the README plugins and include **xdg_directories** as well in this PR? That, or it could possibly be in a separate PR, including any other missing plugins. 😊

**Comment by gspencergoog** [comment]
@jameschensmith Good idea! I forgot that when I added this. Done.

**Pull request merged**